student.rb

```diff
@@ -0,0 +1,21 @@
+require './person'
+
+class Student < Person
+  attr_reader :classroom
+
+  def initialize(classroom, age, name = 'unknown')
+    super(age, name)
```

Comment:
- [ ] **Good job so far 👏 👏** However, it would be better to initialize the *Student* with `parent_permission` argument and call it as a constructor of the *Person* class 😂 

By calling the constructor of the *Person* class with the `parent_permission` argument included, you ensure that the `parent_permission` attribute is initialized in both the *Student* and *Person* classes. This allows you to access the `parent_permission` attribute from both classes and any other subclasses that inherit from *Person*.